### PR TITLE
Highlight codeblocks on hover

### PR DESCRIPTION
**Pull request by plumbis**
This adds support for "hover highlight" in code blocks. A shortcode can be wrapped around a command and on mouse over it will highlight a defined line in a code box. 

Here is an example of the functionality. 
Screen Recording 2022-12-15 at 9 01 24 PM 2022-12-15 21_24_13

This feature is opt-in. Authors must add a `{label="name"}` to a code block and wrap the command in a shortcode, for example, `{{<hover label="name" line="3" >}}MyCommand{{</hover>}}`

Since no docs content has been modified this won't be seen in the Netlify branch preview.

This is a branch of PR #237. This should be merged _after_ that.

### CODE REVIEW

File: utils/webpack/src/js/hoverHighlight.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+// function remove_highlight_line(line_num, line_text, hl_span){
+//     // get the element's parent node
+//     var parent = hl_span.parentNode;
+//     // move all children out of the element
+//     while (hl_span.firstChild) parent.insertBefore(hl_span.firstChild, hl_span);
+//     // remove the empty element
+//     parent.removeChild(hl_span);
+//     line_text.classList.toggle("hl");
+// }
```
Comment (negz) [inline utils/webpack/src/js/hoverHighlight.js:58]: We typically try to avoid checking in commented-out text.

Comment (plumbis) [reply utils/webpack/src/js/hoverHighlight.js]: Removed! I ended up changing how the code block HTML was rendered and left in the v1 method.

File: utils/webpack/src/js/hoverHighlight.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    for (let i = 0; i < keywordList.length; i++) {
+        build_eventlistener(keywordList[i].id, keywordList[i].dataset.label, keywordList[i].dataset.line)
+    }
+}
+
+
+export function build_eventlistener(command_id, code_box_label, code_line_number){
+    // get the dynamic variables created by Hugo shortcode
+    // var command_id = document.currentScript.getAttribute('rand_id');
```
Comment (hasheddan) [inline utils/webpack/src/js/hoverHighlight.js:12]: Do we need to include this commented out code?

Comment (plumbis) [reply utils/webpack/src/js/hoverHighlight.js]: Removed!

**Review by hasheddan — APPROVED** [review]
Thanks @plumbis !

**Comment by plumbis** [comment]
Adds support for "hover highlight" in code blocks.

**Pull request merged**